kano_updater/ui/views/install.py

```diff
@@ -48,10 +50,20 @@ def _create_progress_bar(self):
 
         return self._progress_bar
 
+    def _create_do_not_unplug_label(self):
+        label = Gtk.Label()
+        label.set_text(_("Do not unplug your kit from power or risk data loss."))
+        label.set_size_request(825, 300)
+        label.set_halign(Gtk.Align.CENTER)
+        label.set_valign(Gtk.Align.START)
+        label.get_style_context().add_class('play-game')
```

Comment:
Given that this has nothing to do with the play game label other than it being the same styling, maybe it would be best to rename this CSS class?